### ISSUE TITLE
ICICI bank home page !the radio button under the tab ‘fixed deposit to select the question ‘are you senior citizen  ‘is not announce the value properly.ly

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#4]
UAT 2022-04
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)

NVDA 2021.3.1
Repro Steps
1. Open the URL  https://www.icicibank.com/
2..pressed H for heading for four time to reach the heading ‘104M+ Downloads’ 
.3 after tab two time select the next tab ‘find the right product for you’
4.select deposit in next combo box
5.press tab and then reached to ‘grow my saving’
6. tab to fixed deposit to select the question ‘are you senior citizen’ there are two radio button which is not announce properly 
Expected Result under the tab of fixed deposit to select the question ‘are you senior citizen  -the option should be announced exact selected value like yes/no.

Actual Result .under the tab of ‘fixed deposit to select the question ‘are you senior citizen  screen reader announce ‘radio button checked one of two
WCAG Guideline
1. https://www.w3.org/TR/WCAG21/#name-role-value
